### PR TITLE
media-gfx/pencil: Version Bump to 0.6.4

### PR DESCRIPTION
**Pull request by mundis**
Closes: https://bugs.gentoo.org/684356
Package-Manager: Portage-2.3.81, Repoman-2.3.20
Signed-off-by: Raimund Mathis <iba68@mail.ch>
Package-Manager: Portage-2.3.81, Repoman-2.3.20

### CODE REVIEW

File: media-gfx/pencil/pencil-0.6.1.1.ebuild [hunk 1]
```diff
@@ -40,7 +40,7 @@ src_install() {
 	einstalldocs
 
 	# install target not yet provided
-	# emake INSTALL_ROOT="${D}" install
+	# emake INSTALL_ROOT="${D}" install || die "emake install failed"
```
Comment (flappyports) [inline media-gfx/pencil/pencil-0.6.1.1.ebuild]: emake does not require || die as the eclass handles it. So, the original comment is valid.

File: media-gfx/pencil/metadata.xml [hunk 2]
```diff
@@ -5,4 +5,12 @@
...
 		<remote-id type="sourceforge">pencil-planner</remote-id>
 	</upstream>
+	<maintainer type="person">
+		<email>iba68@mail.ch</email>
+		<name>Raimund Mathis</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```
Comment (juippis) [inline media-gfx/pencil/metadata.xml]: Replace the `<!-- maintainer needed -->` section with this info, ie put this on top of `<upstream>` and remove maintainer-needed.

File: media-gfx/pencil/pencil-0.6.1.1.ebuild [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
```
Comment (juippis) [inline media-gfx/pencil/pencil-0.6.1.1.ebuild]: There's no reason at all for you to even touch this ebuild.

File: media-gfx/pencil/pencil-0.6.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop l10n qmake-utils xdg-utils
```
Comment (juippis) [inline media-gfx/pencil/pencil-0.6.4.ebuild]: You should drop l10n.eclass and install all locales by default, this is now on user to INSTALL_MASK these unwanted small files. I don't really mind it, but it's the right thing to do.

File: media-gfx/pencil/pencil-0.6.4.ebuild [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+DESCRIPTION="2D animation and drawing program based on Qt5"
+HOMEPAGE="https://www.pencil2d.org/"
```
Comment (juippis) [inline media-gfx/pencil/pencil-0.6.4.ebuild]: You could add the Github page here as well.

File: media-gfx/pencil/pencil-0.6.4.ebuild [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+SRC_URI="https://github.com/pencil2d/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+PLOCALES="ca cs da de el es et fr he hu_HU id it ja kab pl pt pt_BR ru sl vi zh_CN zh_TW"
+
+RDEPEND="
+	dev-qt/qtcore:5
```
Comment (juippis) [inline media-gfx/pencil/pencil-0.6.4.ebuild]: You seem to be missing some of the required deps here. You should really study upstream build files to detect how they're used and what are unconditionally required. I see a potential for `doc` USE flag for example, then there's something for `ffmpeg` and at least qtxmlpatterns is required to build this program.

File: media-gfx/pencil/pencil-0.6.4.ebuild [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+	einstalldocs
+
+	newbin bin/pencil2d ${PN}
+
+	newicon app/data/icons/icon.png ${PN}.png
+	make_desktop_entry ${PN} pencil2d ${PN} Graphics
+
+	insinto /usr/share/mime/packages/
+	doins app/data/pencil2d.xml
+	dodoc LICENSE.TXT
```
Comment (juippis) [inline media-gfx/pencil/pencil-0.6.4.ebuild]: No need to duplicate GPL-2 license. Calling `einstalldocs` above is enough.

**Comment by a17r** [comment]
why close it?

**Pull request closed without merging**